<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_interceptorVelocity">
        <property name="text">
         <string>Rocket velocity, m/s</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_interceptorVelocity"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_targetVelocity">
        <property name="text">
         <string>Target velocity, m/s</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_targetVelocity"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_initalDistance">
        <property name="text">
         <string>Initial distance between 
    rocket and target. m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_initialDistance"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pbn_start">
      <property name="text">
       <string>Start simulation</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbn_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startSimulation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startSimulation()</slot>
 </slots>
</ui>
